x/dsmr/block.go

```diff
@@ -119,3 +127,28 @@ type Block struct {
 func (b Block) GetID() ids.ID {
 	return b.blkID
 }
+
+func (b Block) Parent() ids.ID {
```

Comment:
I don't care which we pick, but we should be consistent on the naming of either `Foo()` or `GetFoo()`.

Reply:
I'm going to dodge this one by saying that this method is no longer needed.